NuGet.Config

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
minor nit: can you please tidy up the diff? sometimes just temporarily checking out main and copy pasting the file back into branch can help resolve some of those encoding changes